policyengine_us/variables/household/demographic/person/is_under_court_supervision.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class is_under_court_supervision(Variable):
+    value_type = bool
+    entity = Person
+    label = "Is under court supervision"
+    documentation = "Whether this person is under court supervision."
+    definition_period = YEAR
+    default_value = False
```

Comment:
default default for bools is False
```suggestion
```

Reply:
I think I should remove this variable as we're not using it currently.